src/Transactions.css

```diff
@@ -0,0 +1,27 @@
+li {
```

Comment:
ich würde nicht genrell list items einen style verpassen, außer vllt `list-style: none`.
pack lieber noch eine klasse dazu, sonst kommst du in vllt in probleme wenn du mal ein list item anders haben willst, oder nativ gestylt